src/sagas/staking.js

```diff
@@ -232,6 +232,10 @@ function * getStakingPeriod ({ stakingContract, networkId }) {
   }
 }
 
+function * selectStakingPlatform ({ stakingPlatform }) {
+  yield put(ac)
```

Comment:
The `ac` looks like a typo / mistake, @mul53 Can you check?

Reply:
sorry that's some unfinished code i need to remove